### DISCUSSION TITLE
微信小程序开屏广告

### DISCUSSION THREAD

**Discussion by Aa19139359280** [gkd-kit/gkd#287] [Ideas]
麻烦尽快适配，关闭广告，谢谢了

**Comment by lisonge** [comment]
https://github.com/gkd-kit/subscription/issues

**Reply by Aa19139359280** [reply]
![IMG20231117212825.jpg](https://github.com/gkd-kit/gkd/assets/144342315/4b4052eb-d539-4673-8b6e-ef35d4d97d59)

请问这一种呢？谢谢

**Reply by lisonge** [reply]
https://github.com/gkd-kit/subscription/issues/new/choose
